DataFormats/L1TMuon/src/RegionalMuonCand.cc

```diff
@@ -2,12 +2,11 @@
 
```

Comment:
Hi @abrinke1 I think there is no need to change this file.

Reply:
Why not?  Don't we want the EMTF unpacker and emulator to match, wherever possible?  I confirmed that L1A_BXN() in EMTFBlockSP.cc matches iEvent.bunchCrossing() in MicroGMTConverter.cc.  The offset and truncation is required in both to match how the EMTF firmware fills the track address which is transmitted to the uGMT.

Reply:
It looks as if you just changed the formatting in this file.